### DISCUSSION TITLE
Running 5.0.2787 on docker 20.10.18

### DISCUSSION THREAD

**Discussion by elindberg-snapcode** [datalust/seq-tickets#1663] [Help]
We are long time users of seq and we have for the longest period been trying to maintain an old windows server for this purpose. This, since we have a perpetual pro license and we are very happy with our current version. 
Don't fix what ain't broken, right? :)

However, now we have to give up on this windows server and we want to move over to linux/docker (docker version 20.10.18)
As far as we can understand the latest version of seq we have a license for is 5.0.2787, and luckily enough there is a docker version available.

When starting the stack we get the following error:

>[06:53:31 FTL] Could not "update" config
>System.ArgumentNullException: Value cannot be null.
>Parameter name: config
>   at Seq.Server.Config.Settings.KeyValueSettings.Set(SeqConfig config, String key, String value)
>   at Seq.Server.Cli.ConfigCommand.Run(TextWriter cout)

This is the stack file we are using (I cant seem to fix the indentation)

version: "3.8"

services:
  seq:
    image: datalust/seq:5.0.2787
    ports:
      - 2020:80
      - 5341:5341
    volumes:
      - /data/seq:/data:rw
    environment:
      ACCEPT_EULA: "Y"
      #SEQ_FIRSTRUN_ADMINPASSWORDHASH: "FG+nUnHVw/cpC9Wc11H7l4EdjbWv5GaYZEQJWjK5R3LWOruWYA=="
    deploy:
      placement:
        constraints:
         - node.hostname == scedcr01

As you can see, we have a commented row, and that's because we tried to run the latest version of seq the same way, and that's working out of the box.

Grateful for any pointers on what we can do to fix this?

**Comment by liammclennan** [comment]
Hi Erik,

It's great to hear you are a long term, happy user of Seq!

As far as we can recall it should be possible, but it is hard to be certain as 5.0.2787 has been out of support for a long time. The most reliable method is to first upgrade the windows server to the latest Seq and then migrate to linux/docker.
